docs/technical_reference/property_models/modified_ADM1.rst

```diff
@@ -0,0 +1,307 @@
+Modified ADM1 Property Package
+==============================
+
+This package is an extension of the base ADM1 model and implements properties and reactions of an anaerobic digestion model for industrial wastewater treatment from food and beverage production using an anaerobic digester as provided in
+`Batstone, D. J. et al. (2002) <https://iwaponline.com/wst/article-abstract/45/10/65/6034>`_ and `Rosen and Jeppsson (2006) <https://www.iea.lth.se/WWTmodels_download/TR_ADM1.pdf>`_.
+
+This Anaerobic Digestion Model no.1 (ADM1) makes the following modifications to base ADM1 model as provided in: `Flores-Alsina, X. et al. (2016) <https://www.sciencedirect.com/science/article/pii/S0043135416301397>`_
+   * tracks inorganic phosphorus (S_IP), polyhydroxyalkanoates (X_PHA), polyphosphates (X_PP), phosphorus accumulating organisms (X_PAO), potassium (S_K), and magnesium (S_Mg)
+   * removes the composite material variable (X_C) and the associated disintegration reaction
+   * adds 7 additional reactions
+
+This modified ADM1 property/reaction package:
+   * supports 'H2O', 'S_su', 'S_aa', 'S_fa', 'S_va', 'S_bu', 'S_pro', 'S_ac', 'S_h2', 'S_ch4', 'S_IC', 'S_IN', 'S_IP', 'S_I', 'X_ch', 'X_pr', 'X_li', 'X_su', 'X_aa', 'X_fa', 'X_c4', 'X_pro', 'X_ac', 'X_h2', 'X_I', 'X_PHA', 'X_PP', 'X_PAO', 'S_K', 'S_Mg', 'S_cat', 'S_an', and 'S_co2' as components
+   * supports only liquid and vapor phase
```

Comment:
technically this supports liquid phase and the vapor phase is handled in a separate prop model-- but i guess the doc represents the collective package of liquid/vapor property and reaction models.

Reply:
So should I try to make the distinction more clear here or would that just introduce an unnecessary layer of confusion?

Reply:
Maybe you could mention something indicating that only the liquid phase modeling is modified in this variation of ADM1.